.env.example

```diff
@@ -1,5 +1,5 @@
-POSTGRES_HOST=localhost
```

Comment:
Not sure that this file is needed, so I guess it is better just add this info to the README.md and then remove this file at all

Reply:
Ok